front_end/src/app/shared/services/channel.service.ts

```diff
@@ -180,6 +180,7 @@ export class ChannelService {
                 this.startingSubject.next();
             })
             .fail((error: any) => {
+                alert('Please Refresh to properly connect to server.');
```

Comment:
We do have a story for this but wanted a quick easy solution until we get to that story.